client/termsetting/handlers/samplelst.js

```diff
@@ -118,9 +122,151 @@ export function getSamplelstTW(groups, name = 'groups', groupsetting = {}) {
 		const values = []
 		for (const item of group.items) {
 			const value = { sampleId: item.sampleId }
-			if ('sample' in item) value.sample = item.sample
+			if ('sample' in item) {
+				disabled = false
+				value.sample = item.sample
+			}
 			values.push(value)
 		}
 		return values
 	}
 }
+
+export function showGroupsMenu(event, tw, allowedTermTypes, deleteCallback, app, state) {
+	const samplelstTW = JSON.parse(JSON.stringify(tw))
+	const parentMenu = new Menu({ padding: '5px' })
+
+	const menuDiv = parentMenu.d.append('div')
+
+	let row = menuDiv.append('div')
+
+	addMatrixMenuItems(parentMenu, menuDiv, samplelstTW, app, state)
+	if (allowedTermTypes.includes('survival')) {
+		const survivalDiv = menuDiv
+			.append('div')
+			.attr('class', 'sja_menuoption sja_sharp_border')
+			.html('Compare survival&nbsp;&nbsp;›')
+			.on('click', e => {
+				const state = {
+					nav: { header_mode: 'hide_search' },
+					tree: { usecase: { target: 'survival', detail: 'term' } }
+				}
+				showTermsTree(
```

Comment:
small defect. new menu launched from this option stays persistent and not dismissed, leading to multiple tree menu shown